packages/generator/src/generator-without-ts-morph/service/class.ts

```diff
@@ -46,9 +46,37 @@ export function serviceClass(service: VdmServiceMetadata): string {
     ${service.entities
       .map(entity => getEntityApiFunction(entity, service))
       .join('\n\n')}
+    
+    ${getActionFunctionImports(service, 'functionImports')}
+    
+    ${getActionFunctionImports(service, 'actionImports')}
   }`;
 }
 
+function getActionFunctionImports(
+  service: VdmServiceMetadata,
+  type: 'functionImports' | 'actionImports'
+): string {
+  const names = service[type]
+    ? service[type]!.map(actionOrFunctionImport => actionOrFunctionImport.name)
+    : [];
+
+  if (service[type] === undefined || service[type]!.length === 0) {
+    return '';
+  }
+
+  const lines = service[type]!.map(
+    f =>
+      `${f.name}:(parameter:${f.parametersTypeName}<DeSerializersT>)=>${f.name}(parameter,this.deSerializers)`
+  );
+
+  return codeBlock`
+  get ${type}(){
+  return {${lines.join(',')}}
+  }
+  `;
```

Comment:
General: I know the code goes through a linter later on, but I think it would be nice, if we would apply some formatting ourselved, like spaces between ` => ` and `, `, so that this is also better readable.

Reply:
k